graspologic/utils/utils.py

```diff
@@ -8,7 +8,11 @@
 
 import networkx as nx
 import numpy as np
+from scipy.optimize import linear_sum_assignment
+from sklearn.metrics import confusion_matrix
+from sklearn.metrics._classification import _check_targets
```

Comment:
Since this is a private function in sklearn, we should copy it out into our own code base or rewrite it entirely (depending on licensing details).  We can't guarantee they'll maintain this and relying/using it in our own code will make us more fragile.

Reply:
fair. their license is BSD-3 ([link](https://github.com/scikit-learn/scikit-learn/blob/master/COPYING)). Do you know the answer for that license? If not, I am not sure how to find out with legal.

Reply:
@dwaynepryce or @carolyncb can you advise on this point?

Reply:
update: just need to include in license file -@carolyncb

Reply:
@carolyncb - still think we need to run this by CELA, since we're inlining their code.  pretty sure we just have to maintain a header in the file, but what does this do to the overarching LICENSE.txt for the project? we should just get guidance.